.pre-commit-config.yaml

```diff
@@ -1,4 +1,13 @@
 repos:
+- repo: https://github.com/pycqa/flake8
+  rev: 4.0.1
+  hooks:
+  -   id: flake8
```

Comment:
can we keep same tabulation as black section (also same for blacken-docs)

Reply:
done